pom.xml

```diff
@@ -25,12 +25,9 @@
         <start-class>uk.gov.companieshouse.chd.order.api.ChdOrderApiApplication</start-class>
```

Comment:
The other services put the service name as one word, e.g. item-handler is itemhandler, chd-order-consumer is chdorderconsumer. But I will approve.